### PR TITLE
Feature/configure athena audit logging

### PR DESCRIPTION
**Pull request by melanahammel**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -27,3 +28,4 @@ def setBuildFile(project) {
 rootProject.children.each {
   setBuildFile(it)
 }
+include 'cerberus-audit-logger-athena'
```
Comment (fieldju) [inline settings.gradle]: why is this line here 2x

File: cerberus-web/src/main/java/com/nike/cerberus/event/listener/AuditLoggingEventListener.java [hunk 2]
```diff
@@ -24,8 +24,8 @@
 
 /** Event listener implementation that logs each event. */
 @Slf4j
+@ConditionalOnProperty("cerberus.events.auditLoggingEventListener.enabled")
```
Comment (fieldju) [inline cerberus-web/src/main/java/com/nike/cerberus/event/listener/AuditLoggingEventListener.java]: This should be in the config class that then optionally scans everything in `com.nike.cerberus.audit.athena`
The config class should be in `com.nike.cerberus.config`

**Comment by fieldju** [comment]
Can you port this too: https://github.com/Nike-Inc/cerberus/blob/master/src/test/java/com/nike/cerberus/service/S3LogUploaderServiceTest.java

File: settings.gradle (hunk 1 above)
Comment (melanahammel) [reply settings.gradle]: - [x] remove duplicate line

File: cerberus-web/src/main/java/com/nike/cerberus/config/ApplicationConfiguration.java [hunk 3]
```diff
@@ -48,6 +50,7 @@
 @ComponentScan({
   "com.netflix.spinnaker.kork.secrets",
   "com.nike.backstopper", // error management TODO explicitly import the config bean
+  "com.nike.cerberus.audit.logger",
```
Comment (fieldju) [inline cerberus-web/src/main/java/com/nike/cerberus/config/ApplicationConfiguration.java]: this should be removed and handled in your configuration class.

File: cerberus-web/src/main/java/com/nike/cerberus/event/listener/AuditLoggingEventListener.java (hunk 2 above)
Comment (melanahammel) [reply cerberus-web/src/main/java/com/nike/cerberus/event/listener/AuditLoggingEventListener.java]: - [x] move @ConditionalOnProperty to AthenaAuditLoggerConfiguration

**Comment by melanahammel** [comment]
- [x] port S3LogUploaderServiceTest.java from old project

File: cerberus-web/src/main/java/com/nike/cerberus/config/ApplicationConfiguration.java (hunk 3 above)
Comment (fieldju) [reply cerberus-web/src/main/java/com/nike/cerberus/config/ApplicationConfiguration.java]: example, your config class is in com.nike.cerberus.config, which is already loaded.

then in your config class you have something like the following 

```java
@Configuration
@ConditionalOnProperty("cerberus.userAuth.connector.okta.enabled")
@ComponentScan({"com.nike.cerberus.auth.connector.okta"})
```

Comment (melanahammel) [reply cerberus-web/src/main/java/com/nike/cerberus/config/ApplicationConfiguration.java]: - [x] move @ComponentScan for logger to AthenaAuditLoggerConfiguration
